test/postgresql.test.js

```diff
@@ -820,6 +820,53 @@ describe('Serial properties', function() {
   });
 });
 
+describe('Updating of array properties', function() {
+  let db;
+
+  before(function() {
+    db = global.getSchema();
+  });
+
+  it('should return a valid array', function(done) {
+    const schema =
+      {
+        'name': 'TestWithArray',
+        'properties': {
+          'id': {
+            'type': 'number', 'id': true, 'generated': true,
+          },
+          'productCodes': {
```

Comment:
Can you add `productCodes` as a property of the existing model `Post`?
See its definition in https://github.com/strongloop/loopback-connector-postgresql/blob/dc47d13d5e15d5c31d75fb7c044d2e97542072df/test/postgresql.test.js#L61-L66

And you can add the tests after this one(it also tests patch):
https://github.com/strongloop/loopback-connector-postgresql/blob/dc47d13d5e15d5c31d75fb7c044d2e97542072df/test/postgresql.test.js#L193